core/oio_sds.h

```diff
@@ -384,6 +384,7 @@ struct oio_sds_list_param_s
 
 	char flag_nodeleted : 1;
 	char flag_allversions : 1;
+	char flag_properties : 1;
```

Comment:
It is a good idea to use "unsigned" variable for flag sets, at least to ease further printing/debug effort.
Could you benefit from the current enhancement effort to also turn these existing "signed char" flags into their "unsigned" equivalent?